lib/customer_io_config.dart

```diff
@@ -0,0 +1,47 @@
+import 'customer_io_enums.dart';
+
+/// Configure plugin using CustomerIOConfig
+class CustomerIOConfig {
+  final String siteId;
+  final String apiKey;
+  Region region;
```

Comment:
Region is also optional with a default value. I could be mistaken as this is dart and setting a default value could be done in a different way.